gui/src/app/components/LazyPlotlyPlot.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { FunctionComponent, Suspense } from 'react'
+
+const Plot = React.lazy(() => (import('react-plotly.js')))
+
+type LazyPlotlyPlotProps = {
+	data: any[]
+	layout: any
+}
+
+const LazyPlotlyPlot: FunctionComponent<LazyPlotlyPlotProps> = ({ data, layout }) => {
```

Comment:
```suggestion
import type { PlotParams } from 'react-plotly.js';
const Plot = React.lazy(() => (import('react-plotly.js')))

const LazyPlotlyPlot: FunctionComponent<PlotParams> = ({ data, layout }) => {
```

We can get full typing with this trick (the `type` in `import type` is load-bearing and is what makes this preserve the lazy behavior we desire)